docs/csharp/language-reference/operators/arithmetic-operators.md

```diff
@@ -39,7 +39,13 @@ The following operators perform arithmetic operations with operands of numeric t
 
 Those operators are supported by all [integral](../builtin-types/integral-numeric-types.md) and [floating-point](../builtin-types/floating-point-numeric-types.md) numeric types.
 
-In the case of integral types, those operators (except the `++` and `--` operators) are defined for the `int`, `uint`, `long`, and `ulong` types. When operands are of other integral types (`sbyte`, `byte`, `short`, `ushort`, or `char`), their values are converted to the `int` type, which is also the result type of an operation. When operands are of different integral or floating-point types, their values are converted to the closest containing type, if such a type exists. For more information, see the [Numeric promotions](~/_csharpstandard/standard/expressions.md#1147-numeric-promotions) section of the [C# language specification](~/_csharpstandard/standard/README.md). The `++` and `--` operators are defined for all integral and floating-point numeric types and the [char](../builtin-types/char.md) type.
+Beginning with C# 11, you can define `checked` and unchecked variants of all these operators except the `%` and unary `+` operators. Generally, a `checked` operator is expected to throw an exception when the result overflows the destination type. An unchecked operator generally doesn't throw an exception. The overflow behavior depends on the specific type. Some types, like `int` *truncate*, or wrap around. Other types, like `float` *clamp* to the maximum or minimum value. Still other types, like `decimal` throw exceptions even in an unchecked context. The types in the .NET runtime follow these semantics. The compiler doesn't enforce these requirements, but developers should follow those guidelines in their types. The `checked` operator includes the `checked` keyword. The unchecked operator doesn't include that modifier, as the following example shows:
+
+:::code language="csharp" source="snippets/shared/ArithmeticOperators.cs" id="CheckedOperator":::
```

Comment:
Is this required? I would assume that if you put the `checked` keyword in the operator, you wouldn't need to do it in the code below, because it's only called in the checked context. I've not read the spec, so I'm just pointing out what doesn't make sense.

![image](https://user-images.githubusercontent.com/67293991/173694151-c93bdee2-2efc-459e-a199-a3318711f375.png)

Reply:
Yes. That guarantees that the `operator +` for the `int` type executes in a checked context.